.github/workflows/terrascan.yml

```diff
@@ -9,6 +9,11 @@ on:
         type: string
         default: terraform
         description: e.g. helm, k8s, kustomize, terraform - to see all supported types look at https://github.com/tenable/terrascan-action#iac_type
+      iac_dir:
```

Comment:
This is confusing:
The variable is called iac_dir so I expect it to be `./`, or `terraform/`, but in the default it is `tests`. Why?

Reply:
Your're right. I've changed this.